storm-core/src/jvm/org/apache/storm/task/ShellBolt.java

```diff
@@ -158,8 +158,8 @@ public void execute(Tuple input) {
 
             _pendingWrites.putBoltMsg(boltMsg);
         } catch(InterruptedException e) {
-            String processInfo = _process.getProcessInfoString() + _process.getProcessTerminationInfoString();
-            throw new RuntimeException("Error during multilang processing " + processInfo, e);
+            // It's likely that Bolt is shutting down so no need to throw RuntimeException
```

Comment:
It will be good to have a jvm wide flag to indicate whether jvm shutdown is in progress. There is also a java hack for the same. I can add that flag in a separate PR. Do you mind holding onto this PR till I come up with that?